samples/README.md

```diff
@@ -7,7 +7,7 @@ For a list of available dockerfiles and published images to help with getting st
 * [C#](#C)
 * [C/C++](#CC)
 * [Java](#Java)
-* [Node.js](#Nodejs)
+* [Node.js](./nodejs)
```

Comment:
All links to samples for other language bindings are listed on this page. Any specific reason they shouldn't be aggregated here instead of a readme in the nodejs samples folder? The idea is to have all samples and documented hosted on the web instead of directly included in distributions. CC @natke for any guidance on this.

Reply:
All samples are still aggregated under folder /samples/, and are hosted on web. This change creates a new hierarchy for Node.js without breaking the idea. (reason: I believe most users should be interested in samples of a single language)

If this is not satisfying the requirement, I will remove /samples/nodejs/README.md and merge its content back to /samples/README.md

Reply:
Yes, unless there is a good reason otherwise, I think it should be consistent with the other samples.